doc/nrf/images/thread_platform_design_nRF53.svg

```diff
@@ -0,0 +1,424 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please add the Visio source file as well (for both images).

Reply:
I do not have Visio. I've added Libre Draw sources that can be used to export to any other format accepted by visio.